dockerfile/cargo/Dockerfile

```diff
@@ -226,29 +223,32 @@ LABEL org.opencontainers.image.source="https://github.com/strangelove-ventures/h
 
 WORKDIR /bin
 
-# Install ln (for making hard links), rm (for cleanup), mv, mkdir, vi and dirname from full busybox image (will be deleted, only needed for image assembly)
-COPY --from=busybox-full /bin/ln /bin/rm /bin/mv /bin/mkdir /bin/vi /bin/dirname ./
+# Install dirname
+COPY --from=infra-toolkit /busybox/dirname /bin/dirname
```

Comment:
with this approach, dirname should come from busybox-full and then be removed below.